File: TodayFitComplete_Backend/src/main/java/com/SJY/TodayFitComplete_Backend/dto/member/MemberResponse.java
package com.SJY.TodayFitComplete_Backend.dto.member;

import com.SJY.TodayFitComplete_Backend.entity.member.Member;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * 사용자 정보 반환.
 */
@Getter
@Setter
@NoArgsConstructor
public class MemberResponse {
    private String email;
    private String username;

    @Builder
    public MemberResponse(String email, String username) {
        this.email = email;
        this.username = username;
    }

    public static MemberResponse fromEntity(Member member) {
        return MemberResponse.builder()
                .email(member.getEmail())
                .username(member.getUsername())
                .build();
    }
}

File: TodayFitComplete_Frontend/app/src/main/java/kr/ac/kumoh/ce/s20190633/todayfitcomplete_frontend/Dto/Member/MemberResponse.kt
package kr.ac.kumoh.ce.s20190633.todayfitcomplete_frontend.Dto.Member

// MemberResponse 클래스: 서버에서 클라이언트로 전달되는 사용자 정보를 나타내는 데이터 전송 객체 (DTO)
data class MemberResponse(
    val email: String,    // 사용자 이메일 주소
    val username: String // 사용자의 사용자 이름
)